### PR TITLE
Python Program to find the longest common prefix

### PR DESCRIPTION
**Pull request by narayan2111**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Python Program to find the longest common prefix .

### Checklist

- [Y ] I've read the contribution guidelines.
- [Y ] I've checked the issue list before deciding what to submit.
- [Y ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#917

### CODE REVIEW

**Comment by narayan2111** [comment]
kindly review this PR @iamrajiv @plazzy99 @Kajol-Kumari @Abhijit2505 @atarax665

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@narayan2111 Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

**Comment by narayan2111** [comment]
> @narayan2111 Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

Done Sir @iamrajiv

**Review by iamrajiv — APPROVED** [review]

File: Python/cp/longest_Common_Prefix.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Python Program to find the longest common prefix
+# function to find minimum length
+
+
+def findMinLength(string, size):
```
Comment (Kajol-Kumari) [inline Python/cp/longest_Common_Prefix.py]: @narayan2111  I am getting an error of `IndexError: list index out of range ` in this function. Can you please fix this?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]

File: Python/cp/longest_Common_Prefix.py (hunk 1 above)
Comment (narayan2111) [reply Python/cp/longest_Common_Prefix.py]: > @narayan2111 I am getting an error of `IndexError: list index out of range ` in this function. Can you please fix this?

Yes Ma'am , I fixed this error

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
```
Enter size:3
vatsal
vatsak
vaty
The longest common prefix is  vat
\nSample I/O:\n    Input:\n    Enter size: 3\n    star\n    start\n    stardom\n    Output:\n    The longest Common Prefix is : star\nTime Complexity: O(size*M) (M = Length of the largest string)\nSpace Complexity:O(1)\n
```

correct your output

**Comment by narayan2111** [comment]
> ```
> Enter size:3
> vatsal
> vatsak
> vaty
> The longest common prefix is  vat
> \nSample I/O:\n    Input:\n    Enter size: 3\n    star\n    start\n    stardom\n    Output:\n    The longest Common Prefix is : star\nTime Complexity: O(size*M) (M = Length of the largest string)\nSpace Complexity:O(1)\n
> ```
> 
> correct your output

it works fine on my PC and to make sure i also copied the same code from here and run it then also it's showing correct output

ProgramOutput

File: Python/cp/longest_Common_Prefix.py (hunk 1 above)
Comment (narayan2111) [reply Python/cp/longest_Common_Prefix.py]: > @narayan2111 I am getting an error of `IndexError: list index out of range ` in this function. Can you please fix this?

Yes ma'am I fixed the error. @Kajol-Kumari

**Comment by narayan2111** [comment]
Kindly review this PR @plazzy99 @Kajol-Kumari

**Comment by vatsalkesarwani12** [comment]
> > ```
> > Enter size:3
> > vatsal
> > vatsak
> > vaty
> > The longest common prefix is  vat
> > \nSample I/O:\n    Input:\n    Enter size: 3\n    star\n    start\n    stardom\n    Output:\n    The longest Common Prefix is : star\nTime Complexity: O(size*M) (M = Length of the largest string)\nSpace Complexity:O(1)\n
> > ```
> > 
> > 
> > correct your output
> 
> it works fine on my PC and to make sure i also copied the same code from here and run it then also it's showing correct output
> 
> ProgramOutput

the above output was from google collab

**Comment by narayan2111** [comment]
> > > ```
> > > Enter size:3
> > > vatsal
> > > vatsak
> > > vaty
> > > The longest common prefix is  vat
> > > \nSample I/O:\n    Input:\n    Enter size: 3\n    star\n    start\n    stardom\n    Output:\n    The longest Common Prefix is : star\nTime Complexity: O(size*M) (M = Length of the largest string)\nSpace Complexity:O(1)\n
> > > ```
> > > 
> > > 
> > > correct your output
> > 
> > 
> > it works fine on my PC and to make sure i also copied the same code from here and run it then also it's showing correct output
> > ProgramOutput
> 
> the above output was from google collab

So, how i can fix this error?

**Comment by narayan2111** [comment]
Kindly review this PR @Kajol-Kumari

**Comment by narayan2111** [comment]
Kindly review this PR @Kajol-Kumari @plazzy99

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻

**Comment by narayan2111** [comment]
Kindly review this PR @Abhijit2505

**Comment by narayan2111** [comment]
Kindly review this PR @siddharth25pandey

**Comment by narayan2111** [comment]
Kindly review this PR @Abhijit2505

**Comment by narayan2111** [comment]
Kindly review this PR @HarshCasper

**Pull request merged**